src/components/vms/hostvmslist.jsx

```diff
@@ -42,6 +42,7 @@ import { AggregateStatusCards } from "../aggregateStatusCards.jsx";
 import store from "../../store.js";
 
 import "./hostvmslist.scss";
+import { Alert, List, ListComponent, ListItem, OrderType } from '@patternfly/react-core';
```

Comment:
please move a few lines up and change to use full path import (@patternfly/react-core/dist/esm/components .*), as we import other Patternfly components. There is a reason for that, check the git blame to figure out why if you want :)

Reply:
Got it.

I wonder how reliable the bundler's tree shaking capabilities are.

Reply:
This is the exact reason we moved to using tihs import style. Because importing from react-core would leave some imported but unused CSS in place.